src/components/core/Selector.module.scss

```diff
@@ -0,0 +1,29 @@
+.selector {
+  background-color: #fff;
+  color: black;
+  padding: 8px 10px;
```

Comment:
suggestion: since we are making the selectors bigger, the prev and next buttons are not aligned. Would you please put `align-items: center;` on  `.calendarControls`

Reply:
updated